R/toolEdgeTransportSA.R

```diff
@@ -46,9 +46,6 @@ toolEdgeTransportSA <- function(SSPscen,
   ## Load input data
   ########################################################
   if (is.null(outputFolder) & isStored) stop("Please provide an outputfolder to store your results")
-  if (is.null(gdxPath)) {gdxPath <- file.path(getConfig("sourcefolder"),
```

Comment:
@fbenke-pik 
I think this is the source of the error
`Error in allocateFile(varName) : No subfolder assigned to gdxPath` 
I added the lines again in a local test which fixed the error.
Would you have a look at how this can be reintroduced consistently, or should I?